pkg/image/api/helper.go

```diff
@@ -474,6 +474,8 @@ func ImageWithMetadata(image *Image) error {
 			image.DockerImageLayers[i].Name = layer.DockerBlobSum
 		}
 		if len(manifest.History) == len(image.DockerImageLayers) {
+			// This code does not work since hub.docker.com has been updated.
+			// The V1Compatibility no longer contains Size.
```

Comment:
Rephrase it in following way:
"This code does not work for images imported from v2 schema registries or converted from v2 to v1, since schema v2 does not contain size information."
Or something more appropriate, please.